test/unit/authorize-suite.js

```diff
@@ -108,6 +108,27 @@ define([ 'require', './build/authorize', './build/unauthorized-error'], function
         }
       },
 
+      {
+        desc: "#authorize fails with useful error when there's no scope provided",
+        run: function(env, test) {
+          var authURL = 'http://storage.provider.com/oauth';
+          var scope = undefined;
+          var redirectUri = 'http://awesome.app.com/#custom/path';
+          var clientId = 'http://awesome.app.com/';
+
+          this.localStorageAvailable = function() { return true; };
+
+          var sawError = false;
+          try {
+          env.Authorize.authorize(this, {authURL, scope, redirectUri, clientId});
+          } catch (e) {
+            test.assert(e.message.includes("scope"), true, "Did not throw an error that had a message describing the problem");
+            sawError = true;
+          }
+          test.assert(sawError, true, "Did not throw an error when scope was not set");
```

Comment:
The descriptions of the assertions should be human-readable language of what is being asserted. When the test fails, it will then show that e.g. "throws an error when scope is not set" was expected, but did not satisfy the expectation.

Reply:
Thanks for the feedback! I've updated the assertions' text so that they should match the convention, let me know if not.

Reply:
Great, thanks!